easybuild/tools/filetools.py

```diff
@@ -1166,12 +1167,15 @@ def rmtree2(path, n=3):
 
 def find_backup_name_candidate(src_file):
     """Returns a non-existing file to be used as destination for backup files"""
-    dst_file = src_file
 
-    i = 0
+    # e.g. 20170817234510 on Aug 17th 2017 at 23:45:10
+    timestamp = datetime.datetime.now()
+    dst_file = '%s_%s' % (src_file, timestamp.strftime('%Y%m%d%H%M%S'))
     while os.path.exists(dst_file):
```

Comment:
I think the whole check doesn't make sense now. The chances for the file to already exist are slim at best. But if you end up in that situation (2 instances of EB running concurrently and overwriting each other's files), the backup of the log or of the module file is the least of your worries, unless both of them are `--module-only`